index.html

```diff
@@ -3,6 +3,37 @@
 <html>
   <head>
     <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+<style>
+	  	@media only screen and (min-width: 400px) {
+	body {
+		background: purple;
+		align-content: center;
+	}
+	section {
+		padding: 4rem;
+			}
+}
+		@media only screen and (max-width: 1080px) {
+	body {
+		background: black;
+		align-content: center;
+			}
+	section {
+		height: 150px;
+	}
+	h1 {
+		font-size: 45pt;
+	}
+	h2
+	{
+		font-size: 28pt;	
+	}
+	a:link {
+		font-size: 12pt;		
+			}
+}
+</style>
```

Comment:
Could you move this into your [`index.css` file](https://github.com/Josh-Medley/Josh-Medley_barebones-html-poem/blob/mobliefirst/index.css)?